src/main/webapp/util/UrlConstants.js

```diff
@@ -66,12 +66,17 @@ export const UrlConstants = {
   infoLinkUrl: context + '/api/info-link',
   restrictionUrl: context + '/api/dur',
 
+  submissionsUrl: context + '/api/submissions',
+  submissionsAddNewUrl: '/submissions/add-new',
```

Comment:
Does this not need a context too?

Reply:
No because this url is used to go through the router component.